Makefile

```diff
@@ -31,7 +33,21 @@ fmtcheck: ## Check go formatting
 	@gofmt -l $(SOURCES) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
 
 test: ## Run unit tests
-	@go test ./pkg/...
+	@go test -cover ./pkg/...
+
+test-coverage-html: ## checkout the coverage locally of your tests
+	@echo "mode: count" > coverage-all.out
+	@$(foreach pkg,$(PACKAGES),\
+		go test -coverprofile=coverage.out -covermode=count $(pkg);\
+		tail -n +2 coverage.out >> coverage-all.out;)
+	@go tool cover -html=coverage-all.out # -o coverage.html
+
+ci-test-coverage: ## CI test coverage, upload to coveralls
+	@echo "mode: count" > coverage-all.out
+	@$(foreach pkg,$(PACKAGES),\
```

Comment:
This is a duplicate from `test-coverage-html`, is there a way for you to not repeat yourself?

Reply:
I will look at deduping